program/tests/test_sync_metadata_to_token_2022.rs

```diff
@@ -113,130 +113,20 @@ fn test_fail_wrapped_mint_pda_mismatch() {
         .mint_key(Pubkey::new_unique()) // Not the derived PDA
         .build();
 
-    SyncMetadataBuilder::new()
+    SyncToToken2022Builder::new()
         .wrapped_mint(wrong_wrapped_mint)
         .check(Check::err(TokenWrapError::WrappedMintMismatch.into()))
         .execute();
 }
 
 #[test]
 fn test_fail_wrapped_mint_authority_pda_mismatch() {
-    SyncMetadataBuilder::new()
+    SyncToToken2022Builder::new()
         .wrapped_mint_authority(Pubkey::new_unique()) // Not the derived PDA
         .check(Check::err(TokenWrapError::MintAuthorityMismatch.into()))
         .execute();
 }
 
-#[test]
```

Comment:
The deleted tests here are covered in `program/tests/test_extract_token_metadata.rs` now